site/docs/pages/identity/types.mdx

```diff
@@ -4,3 +4,36 @@ deescription: Glossary of Types in Identity Kit.
 ---
 
 # Types [Glossary of Types in Identity Kit.]
+
+## `EASSchemaUid`
+
+```ts
+type EASSchemaUid = `0x${string}`;
+```
+
+## `EASAttestation`
+
+```ts
+type EASAttestation = {
+  attester: Address; // the attester who created the attestation.
+  decodedDataJson: string; // The attestation data decoded to JSON.
+  expirationTime: number; // The Unix timestamp when the attestation expires (0 for no expiration).
+  id: string; // The unique identifier of the attestation.
+  recipient: Address; // The Ethereum address of the recipient of the attestation.
+  revocationTime: number; // The Unix timestamp when the attestation was revoked, if applicable.
+  revoked: boolean; // A boolean indicating whether the attestation is revocable or not.
```

Comment:
nit `revocable or not` -> `revoked or not`

Reply:
sweet, done!